README.md

```diff
@@ -165,6 +165,69 @@ Depending on the type of the joystick and the personal preference for operation,
 
   > **Note** If for some reason the commands are not passed on to the simulation, try loading, setting and writing the correct parameters for the aircraft using QGC.
 
+Extended Wind Plugin
```

Comment:
Yes, I will probably put the documentation in the wiki (just like for the world generation or camera plugin, as Pavel did).